### ISSUE TITLE
"Fill with module" undo/redo is broken

### ISSUE THREAD

**Issue by DarkFenX** [pyfa-org/Pyfa#2176]
```
[Phobos, Step 1: Import this fit]

Damage Control II
Reactive Armor Hardener
Energized Adaptive Nano Membrane II
[Empty Low slot]
800mm Steel Plates II
Reactor Control Unit II

50MN Microwarpdrive II
Medium F-RX Compact Capacitor Booster
Fleeting Compact Stasis Webifier
Fleeting Compact Stasis Webifier

Warp Disruption Field Generator II
Heavy Neutron Blaster II
Heavy Neutron Blaster II
Heavy Neutron Blaster II
Heavy Neutron Blaster II
Heavy Neutron Blaster II

Medium Polycarbon Engine Housing II
Medium Polycarbon Engine Housing II
```
1) Import the fit
2) Fill with 800mm plate
3) Undo

Result: plate is removed, module positions are shifted, which breaks further undo/redo commands

**Comment by DarkFenX** [comment]
Fixed in f17cf9b736f1199815813c93e38ba9c1c29befc2

**Comment by DarkFenX** [comment]
FYI i fucked up and accidentally reverted the fix in 71ba33edebe928d9fca6ea0c0426f02b9bb9ff0f. Re-implemented it again in 36fd4aeaff1057df076b3bb8662ce68cdc434683
